Add AI Radio plugin documentation (#803)
music-assistant/music-assistant.io: src/content/docs/plugins/ai-radio.md

:::caution[Beta]
AI Radio is currently in beta. The show editor, prompt presets, generated output, and advanced station format may change between Music Assistant releases.
:::

## Features

- Create reusable **shows** from Music Assistant playlists
- Pick a bundled host style such as **Morning show**, **Minimal DJ**, **Music nerd**, or **Party host**
- Choose how often the host speaks with the talkativeness setting
- Generate spoken sections with an AI provider and synthesize them with a TTS provider
- Play a show live through a Music Assistant player
- Generate a completed show and save it as a Music Assistant playlist
- Customize each segment's prompt, timing, web-search behavior, and character limit
- Use placeholders such as `<prev_songinfo>`, `<next_songinfo>`, `<timestamp>`, and `<weather_hourly>`
- Add optional weather updates through Open-Meteo when a weather location is configured
- Duplicate existing shows to create variations without starting from scratch

## Requirements

> - The **AI Radio** plugin must be enabled in **Settings → Providers → Add Provider → Plugins**.

Reviewer: ```suggestion
- The **AI Radio** plugin must be enabled in **Settings → Plugins → Add a Plugin**.
```

Reviewer: Actually you can delete this as it is duplicated below

- At least one playlist with playable tracks.
- A configured provider that supports AI queries, such as the [Home Assistant plugin](/ha-plugin/) with LLM access.
- A configured provider that supports TTS, such as the [Home Assistant plugin](/ha-plugin/) with a working text-to-speech service.
- For live playback, an enabled and available Music Assistant player.

:::note
AI Radio generates new speech each time a show is started. The exact wording can vary even when the same show and playlist are used.
:::

## Installation

> 1. Go to **Settings → Providers → Add Provider**.

Reviewer: ```suggestion
1. Go to **Settings → Plugins → Add a Plugin**.
```

> 2. Open the **Plugins** section.

Reviewer: ```suggestion
```

> 3. Add **AI Radio**.

Reviewer: ```suggestion
2. Add **AI Radio**.
```

> 4. Configure the optional plugin settings:

Reviewer: ```suggestion
3. Configure the optional plugin settings:
```

| Setting | Description |
|---|---|
| **Timezone** | Used for timestamp placeholders and local date formatting. Defaults to the server timezone. |
| **Weather city** | City used when a show segment contains weather placeholders. |
| **Weather country** | Country used together with the city for weather lookup. |

Weather is optional. Shows without weather segments do not need a weather location.

## Setting up AI and TTS with Home Assistant

AI Radio does not contain its own LLM or TTS engine. It asks Music Assistant for providers that expose AI-query and text-to-speech features. The most common setup is to use the [Home Assistant plugin](/ha-plugin/) as the bridge to Home Assistant's Assist/LLM and TTS services.

> Before configuring AI Radio:

Reviewer: This should come before the add plugin step if it needs to be done first

### Weather segments are empty or skipped

Confirm that **Weather city** and **Weather country** are configured in the AI Radio plugin settings. Weather is currently provided by Open-Meteo.

### A customized show warns that it will be simplified on save

Some advanced station configurations cannot be represented perfectly in the simple show editor. Saving rewrites the show into the simplified segment format used by the UI.

## Advanced configuration

Use the AI Radio editor for normal shows. It already supports common timing options, multiple between-song segments, and automatic merging into one natural host break.

Manual JSON editing is only useful when you need behavior the UI cannot describe, such as:

- Customizing the merge prompt used to combine several drafts into one spoken segment
- Combining timing rules in one section, such as "25% chance, max once per hour, and only after five songs"
- Weighted pick-one alternatives

Avoid manual edits for renaming, playlist changes, prompt tweaks, or normal timing. Saving from the UI may simplify advanced settings it cannot represent.

<details>

> <summary>Show Advanced Configuration</summary>

Reviewer: I am not sure we want to add this. Showing people how to get into the MA containers is asking for support requests for broken installations. @MarvinSchenkel what do you think?

Reviewer 2: Yeah let's not advertise these kind of advances things in our docs. Because when people break it, we have to support it.

Reviewer 2: If we want to support this, we could extend the UI for it at a later stage, but we choose to simplify it as much as possible to make it easy to use.
